features/ftue/impl/src/main/kotlin/io/element/android/features/ftue/impl/state/DefaultFtueService.kt

```diff
@@ -119,18 +122,14 @@ class DefaultFtueService @Inject constructor(
                 emit(SessionVerifiedStatus.NotVerified)
             }
             .first()
-        // For some obscure reason we need to call this *before* we check the `readyVerifiedSessionStatus`, otherwise there's a deadlock
-        // It seems like a DataStore bug
-        val skipVerification = canSkipVerification()
-        return readyVerifiedSessionStatus == SessionVerifiedStatus.NotVerified && !skipVerification
+        return readyVerifiedSessionStatus == SessionVerifiedStatus.NotVerified && !canSkipVerification()
```

Comment:
It seems that the previous workaround is not needed anymore CC @jmartinesp

Reply:
Are you sure? Did you try logging out and then in again, and starting the app with no network connection? I'm surprised this issue fixed itself.